apps/nextjs-website/src/lib/strapi/homepage.ts

```diff
@@ -2,12 +2,30 @@ import * as t from 'io-ts/lib';
 import * as tt from 'io-ts-types';
 import * as qs from 'qs';
 import { fetchFromStrapi } from './fetchFromStrapi';
+import { pipe } from 'fp-ts/lib/function';
+import * as E from 'fp-ts/lib/Either';
+
+// converts null to undefined
```

Comment:
It might be helpful to include a comment explaining the circumstances under which it is useful and the rationale behind its creation